.github/workflows/scripts/codepath-notification

```diff
@@ -13,3 +13,7 @@
 # The aim is to find a minimal set of regex patterns that matches any file in these paths
 
 rubicon: header-bidding@magnite.com
+pubmatic: header-bidding@pubmatic.com
+openx: prebid@openx.com
+/adapters/ix|imp_ix|ix.json|ix.yaml: pdu-supply-prebid@indexexchange.com
+medianet: prebid@media.net
```

Comment:
Is this intentionally different from the media net email `prebid-support@media.net` in the yaml file?

Reply:
Not intentional -- prebid@media.net is what's in the java version of the YAML file. https://github.com/prebid/prebid-server-java/blob/382ad292dcd8e4f9e54d8fdc73edba1cacf66688/src/main/resources/bidder-config/medianet.yaml#L2

Will ask them which is correct.

Reply:
Since PBJS has the same, I went ahead and updated the PR.